addons/core/functions/fnc_loadoutReplaceProcess.sqf

```diff
@@ -33,11 +33,8 @@ for [{private _i = (count _loadouts) - 1}, {_i > 0}, {_i = _i - 2}] do {
 
 			// if the item is an actual radio, not a radio prototype nor common item
 			if ((isClass _class) && (isNumber (_class >> "tf_radio"))) then {
```

Comment:
The second part of the if should be in `{ }` for lazy eval.
`if ((isClass _class) && {isNumber (_class >> "tf_radio")}) then {`